mono/metadata/object.c

```diff
@@ -6121,7 +6121,7 @@ mono_string_new_utf16_checked (MonoDomain *domain, const guint16 *text, gint32 l
 
 	MonoString *s;
 	
-	error_init (error);
+	//error_init_internal (error); // FIXME This does not make sense, but fixes test crashes.
```

Comment:
Why not just removing it?

Reply:
Still trying to determine if I need it. I'm revisiting all the newfangled error_init uses, trying to get this PR ready.